case-studies/antipatterns/classic/app/pods/setter-computed-properties/controller.js

```diff
@@ -8,25 +8,25 @@ const Person = EmberObject.extend({
   name: computed('_name', {
     get: function () {
       return this.get('_name').toUpperCase();
-    },
+    },}),
 
-    set: function (_key, nameUpperCase) {
-      this.set('_name', nameUpperCase.toLowerCase());
-      return nameUpperCase;
-    },
-  }),
+  setName(value){
+    this.set("_name", value)
```

Comment:
The idea is to convert user input to an internal representation here.

In my example, the "internal representation" is lowercase, so you could do this:

```js
setName(value){
  this.set('_name', nameUpperCase.toLowerCase());
}
```